build_navitia.sh

```diff
@@ -179,7 +179,7 @@ then
     sudo pip install -r "$navitia_dir"/source/jormungandr/requirements.txt
     sudo pip install -r "$navitia_dir"/source/tyr/requirements.txt
     #we want a custom protobuff version
-    sudo pip install -U protobuf==2.5.0
+    sudo pip install -U protobuf==2.6.1
```

Comment:
This line should just be removed, as it is managed by the requirements (as we want the distrib version)